app/src/main/java/com/example/pocketwatching/Models/Ethplorer/Price.java

```diff
@@ -87,38 +89,6 @@ public void setAvailableSupply(Double availableSupply) {
         this.availableSupply = availableSupply;
     }
 
-    public Double getVolume24h() {
-        return volume24h;
-    }
-
-    public void setVolume24h(Double volume24h) {
-        this.volume24h = volume24h;
-    }
-
-    public Double getVolDiff1() {
-        return volDiff1;
-    }
-
-    public void setVolDiff1(Double volDiff1) {
-        this.volDiff1 = volDiff1;
-    }
-
-    public Double getVolDiff7() {
-        return volDiff7;
-    }
-
-    public void setVolDiff7(Double volDiff7) {
-        this.volDiff7 = volDiff7;
-    }
-
-    public Double getVolDiff30() {
-        return volDiff30;
-    }
-
-    public void setVolDiff30(Double volDiff30) {
```

Comment:
Are you sure you don't need this if you have the getter for this field above?

Reply:
Yes, I can actually remove this variable entirely